### ISSUE TITLE
Secret Agent doesn't run at startup

### ISSUE THREAD

**Issue by linwiz** [maxgoedjen/secretive#555]
I've installed Secretive, and followed the prompt to install Secret Agent, but it does not launch at startup, only when I manually open Secretive. This causes ssh connections to fail instantly. (ssh: connect to host xxxxx port 22: Undefined error: 0)

I have tried running the 'Setup Secretive' several times, but every time I reboot my machine, Secret Agent does not run automatically. I see no possible option to add Secret Agent manually to my 'login items' and am at a loss of what to do next.

**Comment by tiehfood** [comment]
I have this problem too. This is what I get, if I try to run the agent from the terminal:
`SecretAgent[24627:137916] WARNING: The sun_len field of a sockaddr_un structure passed to CFSocketSetAddress was not set correctly using the SUN_LEN macro.
SecretAgent[24627:137916] CFSocketSetAddress bind failure: 48`

**Comment by tiehfood** [comment]
Any chance to get this fixed? I really liked the app on the previous devices :)

**Comment by maxgoedjen** [comment]
Are you folks seeing this consistently every reboot? I'm aware of some ongoing (hard to debug) issues around it relaunching after an update, but not so much every launch.

**Comment by tiehfood** [comment]
I think on the first run I shortly saw the Agent on the Activity Monitor. It then exited. After the first run the Agent never appeared again on the Activity Monitor and if I try to run it, I will get that error message. But I could run it as root (but then it does not work either)

**Comment by tiehfood** [comment]
I closed the app multiple times completely and also did a reboot

**Comment by maxgoedjen** [comment]
Definitely shouldn't require root. If you open up macOS settings and look at "Login Items," do you see Secretive there (and is it on?) 
<img width="827" alt="image" src="https://github.com/maxgoedjen/secretive/assets/342665/f79ae27c-a94c-4cd8-afa1-47fa008d9741">

**Comment by tiehfood** [comment]
yes, it's there and activated

**Comment by tiehfood** [comment]
Any news?

**Comment by linwiz** [comment]
Yes, this is ongoing and there is a login item for secretive, and it's enabled. but it does not run (or doesn't stay running; either way) this is still an issue

**Comment by linwiz** [comment]
It appears that it has started working, that is, app now starts when macOS restarts. I am not sure of what changes if any were made.
